frontend/packages/ux-editor/src/containers/FormDesignerToolbar.tsx

```diff
@@ -3,11 +3,51 @@ import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmen
 import classes from './FormDesignerToolbar.module.css';
 import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
 import { LayoutSetsContainer } from '../components/Elements/LayoutSetsContainer';
+import { FeedbackForm } from './DesignView/AddItem/FeedbackForm';
+import {
+  addFeatureFlagToLocalStorage,
+  removeFeatureFlagFromLocalStorage,
+  shouldDisplayFeature,
+} from 'app-shared/utils/featureToggleUtils';
+import { StudioParagraph, StudioSwitch } from '@studio/components';
+import { HelpText } from '@digdir/designsystemet-react';
 
 export const FormDesignerToolbar = () => {
   const { org, app } = useStudioEnvironmentParams();
   const layoutSetsQuery = useLayoutSetsQuery(org, app);
   const layoutSetNames = layoutSetsQuery?.data?.sets;
 
-  return <section className={classes.toolbar}>{layoutSetNames && <LayoutSetsContainer />}</section>;
+  const toggleComponentModalPocAndReload = () => {
+    if (shouldDisplayFeature('addComponentModal')) {
+      removeFeatureFlagFromLocalStorage('addComponentModal');
+    } else {
+      addFeatureFlagToLocalStorage('addComponentModal');
+    }
+    window.location.reload();
+  };
+
+  return (
+    <section className={classes.toolbar}>
+      {layoutSetNames && <LayoutSetsContainer />}
+      <div style={{ display: 'flex', flexDirection: 'row', gap: 'var(--fds-spacing-2)' }}>
+        <StudioSwitch
+          checked={shouldDisplayFeature('addComponentModal')}
+          onChange={toggleComponentModalPocAndReload}
+        >
+          Prøv nytt design
```

Comment:
Maybe we should add a comment in the component why we use hardcoded texts and not text keys?😊

Reply:
Yep - I ended up just creating a new component for all of this functionality, and adding a comment in the component file explaining why the explicit texts are used.